### PR TITLE
Fix flash messages error

### LINKED ISSUE #13: Error message on open login page
When open a login page, a message containing { } appears.

If user inputs an invalid value the following message appears: {"alert"=>"Email ou senha inválidos."} .

### LINKED ISSUE #14: Error message on open event page
when open an event page a message containing { } appears.

### PR DESCRIPTION
**Pull request by gmmcal**
fixes #13 and fixes #14
closes #13 and closes #14

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://gmmcal-fix-flash-messages.alagoasdevday-call4papers-3062b296e4351a66c3e5.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://gmmcal-fix-flash-messages.alagoasdevday-call4papers-3062b296e4351a66c3e5.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -9,15 +9,11 @@ def bootstrap_class_for(flash_type)
...
-      concat(content_tag(:div, message,
-                         class: "alert #{bootstrap_class_for(msg_type)} fade in"
-        ) do
-          concat content_tag(:button, 'x',
-                             class: 'close', data: { dismiss: 'alert' })
-          concat message
-        end
-      )
+      flash_messages <<
+        "<div class=\"alert #{bootstrap_class_for(msg_type)} fade in\"><button data-dismiss=\"alert\" class=\"close\">x</button>#{message}</div>"
```
Comment (pragmaticivan) [inline app/helpers/application_helper.rb]: Oo?

Comment (pragmaticivan) [inline app/helpers/application_helper.rb]: You can solve this Pretty feature (lol) with a partial hahah.

Comment (gmmcal) [inline app/helpers/application_helper.rb]: will do. thanks

**Comment by gmmcal** [comment]
done

**Pull request merged**